StripeCore/StripeCore/Source/Localization/STPLocalizationUtils.swift

```diff
@@ -8,7 +8,7 @@
 
 import Foundation
 
-@_spi(STP) public final class STPLocalizationUtils {
+public final class STPLocalizationUtils {
```

Comment:
Why is this (and BundleLocatorProtocol) public now?

Reply:
Great catch! These changed when I was trying to see why and objc file wouldn't recognize them but that file ended up being unused and removed, and I forgot to revert these, reverted now.